ArcadiaParties-SPA/src/app/header/header.component.html

```diff
@@ -1,4 +1,3 @@
-<div>{{ title$ | async }}</div>
+<a class="cursor" [routerLink]="['/home']">{{ title$ | async }}</a>
```

Comment:
`a` element doesn't need to override cursor styles.

Reply:
Class deleted